deploy/chart/webhook/templates/ca-sync.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
-  schedule: "* * */24 * *"
+  schedule: "@weekly"
```

Comment:
can you also update the comment at the top of the file about the fact it's now weekly?

Reply:
Ah yes that makes sense, leave it with me.